iso15118/secc/states/iso15118_2_states.py

```diff
@@ -1554,6 +1565,32 @@ async def process_message(
 
         self.expecting_power_delivery_req = False
 
+    async def wait_for_state_c_or_d(self) -> bool:
+        STATE_C_TIMEOUT = 0.25
```

Comment:
Would be good to mention [V2G2-847] as the reason for 0.25.

Reply:
Done